### PR TITLE
Further design fixes

### PR DESCRIPTION
**Pull request by jancborchardt**
- Fix 'Add question' button overlapping navigation toggle
- Improve vertical spacing
- Fix answer placeholder for long and short answer question
- Remove max-height for description so it is always shown in full 

## Before
All jumbled and spacing not good
before questions
before

## After
All fixed
spacing
Overlap and question types

### CODE REVIEW

File: src/components/Questions/QuestionLong.vue [hunk 1]
```diff
@@ -30,8 +30,8 @@
 		<div class="question__content">
 			<!-- TODO: properly choose max length -->
 			<textarea ref="textarea"
-				:aria-label="t('forms', 'A long answer for the question “{text}”', { text })"
-				:placeholder="t('forms', 'Long answer text')"
+				:aria-label="t('forms', 'People can enter a long answer for the question “{text}”', { text })"
```
Comment (skjnldsv) [inline src/components/Questions/QuestionLong.vue:33]: Not sure this is what aria expects. Aria is more like instructions, so I would keep something describing what I (as a person) have to do, not what others can do ?

Comment (jancborchardt) [reply src/components/Questions/QuestionLong.vue:33]: Thing is, as a survey creator, this field is useless. It does not do anything, which is why we said we fix the placeholder. ;)

Comment (jancborchardt) [reply src/components/Questions/QuestionLong.vue]: Rebased and fixed this to not change the aria value.

File: src/views/Submit.vue [hunk 2]
```diff
@@ -153,6 +153,11 @@ export default {
 	align-items: center;
 	flex-direction: column;
 
+	// Force hide navigation toggle as there is no navigation
+	::v-deep #app-navigation-toggle {
+		display: none !important;
+	}
```
Comment (skjnldsv) [inline src/views/Submit.vue:159]: Will be fixed with upcoming release of vue components.

File: src/views/Submit.vue [hunk 3]
```diff
@@ -163,35 +168,49 @@ export default {
...
+
+		::v-deep ul.question__content {
+			// Left-align multiple choice and checkboxes with question text
+			margin-left: -14px;
+		}
+
+		// Overwrite core styles
+		::v-deep .question__label {
+			text-align: left !important;
+		}
```
Comment (skjnldsv) [inline src/views/Submit.vue:208]: Why here and not in the appropriate component?
https://github.com/nextcloud/forms/blob/7462be7bfedc8a2d55cdd5ad5a2fa36dfc8dc121/src/components/Questions/Question.vue#L194
https://github.com/nextcloud/forms/blob/7462be7bfedc8a2d55cdd5ad5a2fa36dfc8dc121/src/components/Questions/Question.vue#L194

Comment (skjnldsv) [reply src/views/Submit.vue:208]: Let me fix so we can do merge this

Comment (skjnldsv) [reply src/views/Submit.vue]: Ah ok, I get it now!
I only moved the text align then as it make sense for submit & create view :+1: 
Great work!

Comment (jancborchardt) [reply src/views/Submit.vue]: Yeah, as the issue only appears in body-public.

**Pull request merged**